src/FluentMigrator.Runner.SQLite/Generators/SQLite/SQLiteColumn.cs

```diff
@@ -35,7 +68,7 @@ protected override string FormatIdentity(ColumnDefinition column)
                 // see: http://www.sqlite.org/syntaxdiagrams.html#column-constraint syntax details
                 if (!column.IsPrimaryKey && (!column.Type.HasValue || GetTypeMap(column.Type.Value, null, null) != "INTEGER"))
                 {
-                    throw new ArgumentException("SQLite only supports identity on a single integer, primary key coulmns");
+                    throw new ArgumentException("SQLite only supports identity on a single integer, primary key coulmn");
```

Comment:
I wonder if we should also format this message to include the column name from `ColumnDefinition column` so that there is some context.  Not critical.

Reply:
Updated comment 👍